### PR TITLE
Added support for ImportToken

### PR DESCRIPTION
**Pull request by fofiuiancu**
**Added**

- ImportToken to Session Specification Builder and a ImportTokenBuilder

**Requested with**
```go	
ttl := time.Hour * 24 * 30
importToken, err := create.NewImportTokenBuilder().
	WithTTL(int(ttl.Seconds())).
	Build()
if err != nil {
	return nil, err
}

sessionSpec, err = create.NewSessionSpecificationBuilder().
		WithClientSessionTokenTTL(6000).
		WithResourcesTTL(900000).
		WithUserTrackingID("some-tracking-id").
		WithSDKConfig(sdkConfig).
		WithIdentityProfileRequirements(identityProfile).
		WithCreateIdentityProfilePreview(true).
		WithSubject(subject).
                WithImportToken(importToken).
		Build()
```

- ImportTokenResponse to IDV GetSessionResult:

```go
getSessionResult, err := client.GetSession(sessionId)
if err != nil {
        return err
}
importTokenResponse := getSessionResult.ImportTokenResponse
```

Example available when running the idv example project at https://localhost:8080/dbs

### CODE REVIEW

File: docscan/session/create/import_token.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package create
+
+import "time"
+
+type ImportToken struct {
+	Ttl int `json:"ttl"`
+}
+
+// defaultImportTokenTTL is 12 Months
+const defaultImportTokenTTL = time.Hour * 24 * 30 * 12
```
Comment (klaidas) [inline docscan/session/create/import_token.go:10]: 12 months seems like a very long time, i'd make the default 1 month so we're recommending security (by giving an attacker less time) and only the RP can change it if they need it to be longer.

**Review by mehmet-yoti — APPROVED** [review]

File: docscan/session/create/session_spec.go [hunk 2]
```diff
@@ -140,6 +144,12 @@ func (b *SessionSpecificationBuilder) WithSubject(subject json.RawMessage) *Sess
 	return b
 }
 
+// WithImportToken sets whether of not an ImportToken is to be generated.
```
Comment (klaidas) [inline docscan/session/create/session_spec.go:147]:
```suggestion
// WithImportToken sets whether or not an ImportToken is to be generated.
```

**Review by klaidas — APPROVED** [review]
LGTM

File: docscan/session/create/import_token.go (hunk 1 above)
Comment (klaidas) [reply docscan/session/create/import_token.go:10]: oh also i wouldn't say that's accurately 12 months, if you wanna do a year i would've just done `time.Hour * 24 * 365` (rip leap years)

Comment (fofiuiancu) [reply docscan/session/create/import_token.go:10]: in the ticket we're defining ttl as: "seconds, not null, default to 12 months, MAX 12 months, not less than 30 days"

**Review by klaidas — APPROVED** [review]

**Pull request merged**